src/ApiConnectors/ArticleApiConnector.php

```diff
@@ -59,9 +59,7 @@ public function send(Article $article): Article
 
         Assert::count($articleResponses, 1);
 
-        foreach ($articleResponses as $articleResponse) {
-            return $articleResponse->unwrap();
-        }
+        return $articleResponses->getIterator()->current()->unwrap();
```

Comment:
This should be moved to a `final protected method unwrapSingleResponse(array $responses)` on the parent class.

Reply:
✅ Added the method to the base class and used it.